src/JsonService.ts

```diff
@@ -74,7 +98,7 @@ export class JsonService {
         return json;
     }
 
-    public async postForm(url: string, body: URLSearchParams, basicAuth?: string): Promise<Record<string, unknown>> {
+    public async postForm(url: string, body: URLSearchParams, basicAuth?: string, timeoutInSeconds?: number): Promise<Record<string, unknown>> {
```

Comment:
This class is internal so we can always come back to this later, but just want to criticize that this function now takes in 4 parameters 😬

Reply:
done in separate MR, which can be merged after this one. See #349